src/Snak/SnakObject.php

```diff
@@ -97,12 +97,12 @@ public function equals( $target ) {
 	/**
 	 * @see Serializable::serialize
 	 *
-	 * @since 0.1
+	 * @since 7.0
```

Comment:
We have some places where we bumped the number but then mention the method was already there in some other form or something along those lines